op-mode-definitions/monitor-bandwidth-test.xml.in

```diff
@@ -7,21 +7,50 @@
           <help>Initiate or wait for bandwidth test</help>
         </properties>
         <children>
-          <leafNode name="accept">
+          <node name="accept">
```

Comment:
The old `accept` node simply called: `<command>/usr/bin/iperf -V -s</command>`, can we make this backwards compatible and behave in the same way?

Reply:
Please check if my amendment is correct. I'm not sure if I understand your statement?